CHANGES.md

```diff
@@ -4,3 +4,7 @@
   + Add `check` and `fix` command.
   + `check` results can be reported to pull request as comments.
   + `fix` suggestions can be filed on pull request as edit suggestions.
+
+- Dependencies License Report
+  + Support Golang `go.mod` dependency report.
+  + Licences Apache 2.0, BSD-3-Clause and MIT can be identified.
```

Comment:
Also want to ask for @wu-sheng your opinion on, whether we want to include this "Dependencies License Report" in the first version, or just don't mention this in the change log until we have more complete supported list of languages / licenses, maybe next version?

Reply:
I prefer to skip the unstable features.